index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Matthew Wilson's Portfolio</title>
+</head>
+<body>
+  <h1>Matthew Wilson's Web Development Portfolio</h1>
+  <p>Check out some of the great projects I've been working on:</p>
+  <section id="projects">
+  <section class="title"></section>
+  <section class="date"></section>
+  <section class ="description"></section>
+  <section class ="link"></section>
+</section>
```

Comment:
Where's jQuery? You need to include the file if you're going to use it. Either download and put it in your repo or add a link to a CDN.